From 65c816a79da47e034bc4f0dc6b59538964b3ae6e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6nke=20Behrendt?= <thesoenke@outlook.com>
Date: Sat, 7 Jan 2017 15:11:22 +0100
Subject: [PATCH 1/1] Change default plugin directory

Use /usr/share/OFX instead of /usr/OFX as the default plugin directory
---
 Engine/Settings.cpp                             | 2 +-
 libs/OpenFX/HostSupport/src/ofxhPluginCache.cpp | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Engine/Settings.cpp b/Engine/Settings.cpp
index c55235b..787ef9d 100644
--- a/Engine/Settings.cpp
+++ b/Engine/Settings.cpp
@@ -1302,7 +1302,7 @@ Settings::initializeKnobsPlugins()
     _extraPluginPaths->setName("extraPluginsSearchPaths");
 
 #if defined(__linux__) || defined(__FreeBSD__)
-    std::string searchPath("/usr/OFX/Plugins");
+    std::string searchPath("/usr/share/OFX/Plugins");
 #elif defined(__APPLE__)
     std::string searchPath("/Library/OFX/Plugins");
 #elif defined(WINDOWS)
diff --git a/libs/OpenFX/HostSupport/src/ofxhPluginCache.cpp b/libs/OpenFX/HostSupport/src/ofxhPluginCache.cpp
index 80c32f2..70875a8 100644
--- a/libs/OpenFX/HostSupport/src/ofxhPluginCache.cpp
+++ b/libs/OpenFX/HostSupport/src/ofxhPluginCache.cpp
@@ -284,7 +284,7 @@ PluginCache::PluginCache() : _hostSpec(0), _xmlCurrentBinary(0), _xmlCurrentPlug
   _pluginPath.push_back("C:\\Program Files\\Common Files\\OFX\\Plugins");
 #endif
 #if defined(__linux__) || defined(__FreeBSD__)
-  _pluginPath.push_back("/usr/OFX/Plugins");
+  _pluginPath.push_back("/usr/share/OFX/Plugins");
 #endif
 #if defined(__APPLE__)
   _pluginPath.push_back("/Library/OFX/Plugins");
@@ -300,7 +300,7 @@ void PluginCache::setPluginHostPath(const std::string &hostId) {
   _pluginPath.push_back("C:\\Program Files\\Common Files\\OFX\\" + hostId);
 #endif
 #if defined(__linux__) || defined(__FreeBSD__)
-  _pluginPath.push_back("/usr/OFX/" + hostId);
+  _pluginPath.push_back("/usr/share/OFX/" + hostId);
 #endif
 #if defined(__APPLE__)
   _pluginPath.push_back("/Library/OFX/" + hostId);
-- 
2.11.0

